### ISSUE TITLE
Region Change

### ISSUE THREAD

**Issue by Vegatta** [Plailect/Guide#460]
I have recently performed a region transfer of a old 3ds Japan to Europe. I used the ctr transfer from the guide. After the transfer, whenever I enter a app (like the manual, notification, homebrewmanager)  and then go back to home menu; the console freezes at the home menu banner and I have to perform a hard reset.

Has anyone an idea how I can do to fix that, or is it maybe a error in Luma?

**Comment by jaxrogers2** [comment]
Make sure that 11.0 Menuhax is not installed- that will cause that issue because of the way Menuhax is stored in 10.6+.

**Comment by Vegatta** [comment]
No, I had no Menuhax on it. The 3ds was new and only downgraded via game and than I followed the guide. A test on a second 3ds had the same result. Actually I figured out that a format of the 3DS after the region change fixed the problem. I guess somewhere there where still information from the Japanese menu saved beside me deleting the folder 00000082 as a test before. So maybe as toubleshoot idea just add the point "If there is a problem with return to home menu, try format (yes its safe)" or something like this. After all people discussed the same problem on reddit too. I actually got the idea with format from their discussion.
